internal/v2/loadbalancer.go

```diff
@@ -0,0 +1,40 @@
+package v2
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// UnmarshalJSON unmarshals a LoadBalancer resource into a temporary structure whose "CreatedAt" field of type
+// string to be able to parse the original timestamp (ISO 8601) into a time.Time object, since json.Unmarshal()
+// only supports RFC3339 format.
```

Comment:
Alternatively, you could have created your own `ISO8601Time`, https://github.com/exoscale/egoscale/blob/master/uuid.go#L44-L61